Missing comma in param block

Missing comma in the param block

https://github.com/rfennell/AzurePipelines/blob/89989eb27fbbabc87881604665f90cd526d5c47f/Extensions/Versioning/VersionSharepointAppManifestTask/ApplyBuildVersionToSharepointAppManifest.ps1#L23

Causes error:
```
2018-12-14T12:18:49.4242817Z ##[error]System.Management.Automation.ParseException: At F:\AGENT1\_work\_tasks\VersionSharepointAppManifest_2d932e86-7068-4a1e-ab4d-f104fad6db37\2.3.21\ApplyBuildVersionToSharepointAppManifest.ps1:23 char:19
2018-12-14T12:18:49.4243149Z ##[debug]Processed: ##vso[task.logissue type=error;]System.Management.Automation.ParseException: At F:\AGENT1\_work\_tasks\VersionSharepointAppManifest_2d932e86-7068-4a1e-ab4d-f104fad6db37\2.3.21\ApplyBuildVersionToSharepointAppManifest.ps1:23 char:19
2018-12-14T12:18:49.4243441Z +     $InjectVersion
2018-12-14T12:18:49.4243502Z +                   ~
2018-12-14T12:18:49.4243551Z Missing ')' in function parameter list.
2018-12-14T12:18:49.4243578Z 
2018-12-14T12:18:49.4243624Z At F:\AGENT1\_work\_tasks\VersionSharepointAppManifest_2d932e86-7068-4a1e-ab4d-f104fad6db37\2.3.21\ApplyBuildVersionToSharepointAppManifest.ps1:28 char:1
2018-12-14T12:18:49.4243661Z + )
2018-12-14T12:18:49.4243708Z + ~
2018-12-14T12:18:49.4243747Z Unexpected token ')' in expression or statement.
2018-12-14T12:18:49.4243788Z    at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
2018-12-14T12:18:49.4243838Z    at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
2018-12-14T12:18:49.4243894Z    at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
2018-12-14T12:18:49.4243942Z    at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2018-12-14T12:18:49.4244765Z    at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
2018-12-14T12:18:49.4244814Z    at Microsoft.TeamFoundation.DistributedTask.Handlers.LegacyVSTSPowerShellHost.VSTSPowerShellHost.Main(String[] args)
2018-12-14T12:18:49.4347667Z ##[error]LegacyVSTSPowerShellHost.exe completed with return code: -1.
```

--- a/Extensions/Versioning/VersionSharepointAppManifestTask/ApplyBuildVersionToSharepointAppManifest.ps1
+++ b/Extensions/Versioning/VersionSharepointAppManifestTask/ApplyBuildVersionToSharepointAppManifest.ps1
@@ -20,7 +20,7 @@ param (
     [Parameter(Mandatory)]
     [string]$VersionNumber,
 
-    $InjectVersion
+    $InjectVersion,
     
     [string]$VersionRegex,
 

--- a/Extensions/Versioning/readme.md
+++ b/Extensions/Versioning/readme.md
@@ -95,3 +95,4 @@ The Android manifest versioner takes the following extra parameters:
 - V2.1.x  - Fix Android Versioning task from assigning a 0 to VersionCode, build will now fail if it is.
 - V2.2.x  - Issue398 fix for non Windows Platforms for tasks that run cross platform 
 - V2.3.x  - Issue306 added ability to inject a version number directly (no regex processing) to all tasks
+- V2.4.x  - Issue406 adding back in missing comma from parameter block in SharePoint task.

